inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/transformation/relation/LeftOuterJoinNodeRelationTest.java

```diff
@@ -38,10 +38,15 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * Tests for{@link LeftOuterJoinNodeRelationTest}
+ * Tests for {@link LeftOuterJoinNodeRelationTest}
```

Comment:
`LeftOuterJoinNodeRelationTest` should be LeftOuterJoinNodeRelation

Reply:
done